testsuite/python/lb_couette_xy.py

```diff
@@ -0,0 +1,147 @@
+#
+# Copyright (C) 2021-2023 The ESPResSo project
+#
+# This file is part of ESPResSo.
+#
+# ESPResSo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ESPResSo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import espressomd.lb
+import espressomd.lees_edwards
+
+import unittest as ut
+import unittest_decorators as utx
+import numpy as np
+
+
+def analytical(x, t, nu, v, h, k_max):
+    """
+    Analytical solution with Fourier series of the Navier-Stokes equation.
+
+    Parameters
+    ----------
+    x : :obj:`float`
+        Height within the channel
+    t : :obj:`float`
+        Time since the start up of the shear flow
+    nu: :obj:`float`
+        Kinematic kinematic_viscosity
+    v: :obj:`float`
+        Shearing velocity
+    h : :obj:`float`
+        Distance between shear planes
+    k_max : :obj:`int`
+        Upper limit of sums for sinus series
+
+    """
+    u = x / h - 0.5
+    for k in np.arange(1, k_max + 1):
+        wave = 2 * np.pi * k / h
+        u += np.exp(-nu * wave ** 2 * t) * np.sin(wave * x) / (np.pi * k)
+    return v * u
+
+
+LB_PARAMS = {'agrid': 1.,
```

Comment:
Pls avoid 1s in unit tests, as wrong exponents don't get cauth.

Reply:
I moved this function.

Reply:
I changed the value of agrid from 1. to 0.6.